Tasks/MavenV3/maventask.ts

```diff
@@ -565,4 +568,35 @@ function execBuildWithRestore() {
         execBuild();
     }
 }
+
+// function read code coverage report as DOM
+function readCodeCoverageReportAsDom(): jsdom.JSDOM {
+    const htmlString: string = fs.readFileSync(path.join(reportDirectory, 'index.html'), 'utf-8');
+    return new JSDOM(htmlString);
+}
+
+// function write DOM as html
+function writeDomAsHtml(dom: jsdom.JSDOM): void {
+    fs.writeFileSync(path.join(reportDirectory, 'index.html'), dom.serialize())
+}
+
+// function replace images to base64 code in Code Coverage report html
+function replaceGifToBase64(): void {
+    try {
+        const dom = readCodeCoverageReportAsDom();
+        const images: HTMLImageElement[] = [...dom.window.document.getElementsByTagName('img')];
+        images.forEach(element => {
+            const pathToImg: string = path.join(reportDirectory, element.src)
+            if(fs.existsSync(pathToImg) && fs.statSync(pathToImg).size/1024 < 1024) {
```

Comment:
Please move the limitation out as constant and make it more readable

Reply:
Done.